app/src/main/java/org/fossasia/susi/ai/chat/ChatPresenter.kt

```diff
@@ -383,7 +383,12 @@ class ChatPresenter(chatActivity: ChatActivity) : IChatPresenter, IChatModel.OnR
                     psh.parseSusiResponse(susiResponse, i, utilModel.getString(R.string.error_occurred_try_again))
 
                     var setMessage = psh.answer
-                    if (psh.actionType == Constant.ANSWER && (PrefManager.checkSpeechOutputPref() && check || PrefManager.checkSpeechAlwaysPref())) {
+                    if (psh.actionType == Constant.VIDEOPLAY || psh.actionType == Constant.AUDIOPLAY) {
+                        // Play youtube video
+                        Timber.d("Playing a youtube video")
+                        return@postDelayed
+
```

Comment:
Remove extra line

Reply:
The code in this area is pretty congested so I had to add extra line for better visibility, hope you can relate :)

Reply:
I feel that is quite subjective! 😄 
imho it should be removed. Rest looks fine. :)

Reply:
@arundhati24 it has to be changed in the next PR, so I don't think it is blocking any app feature or not following any of the code constructs, if this is not an issue at the moment please merge this so that I can work on the next PR. :)

Reply:
I don't think it was a very time taking change to do. Anyhow, okay. Please make sure to remove that in the next one. :)